server/backend/tests/tests/crud/test_crud_utils.py

```diff
@@ -42,35 +42,35 @@ def test_get_next(dbsession_fam_users, userdata2_pydantic, delete_all_users):
     assert next_value_after > next_value_before
 
 
-@pytest.mark.parametrize("str_list_to_test, expcted_str_list",[
-    (['fam', 'fom', 'aws'], ['FAM', 'FOM', 'AWS']),
-    (['FAM', 'FOM', 'AWS'], ['FAM', 'FOM', 'AWS']),
-    (None, None)
-])
-def test_to_upper(str_list_to_test, expcted_str_list):
-    result = crud_utils.to_upper(str_list_to_test)
-    if result:
-        for index, r in enumerate(result):
-            assert r == expcted_str_list[index]
-    else:
-        assert result == expcted_str_list
+# @pytest.mark.parametrize("str_list_to_test, expcted_str_list",[
+#     (['fam', 'fom', 'aws'], ['FAM', 'FOM', 'AWS']),
+#     (['FAM', 'FOM', 'AWS'], ['FAM', 'FOM', 'AWS']),
+#     (None, None)
+# ])
+# def test_to_upper(str_list_to_test, expcted_str_list):
+#     result = crud_utils.to_upper(str_list_to_test)
+#     if result:
+#         for index, r in enumerate(result):
+#             assert r == expcted_str_list[index]
+#     else:
+#         assert result == expcted_str_list
 
 
-@pytest.mark.parametrize("str_list_to_test, str_to_replace, replace_with, expcted_str_list",[
-    (['FAM_ACCESS_ADMIN', 'FOM_DEV_ACCESS_ADMIN', 'FOM_TEST_ACCESS_ADMIN'], 
-     "_ACCESS_ADMIN", "",
-     ['FAM', 'FOM_DEV', 'FOM_TEST']
-    ),
-    (['FAM_ACCESS', 'FOM_DEV', 'FOM'], 
-     "_ACCESS", "_ACCESS_ADMIN",
-     ['FAM_ACCESS_ADMIN', 'FOM_DEV', 'FOM']
-    ),
-    (None, "something", "some_other_thing", None)
-])
-def test_replace_str_list(str_list_to_test, str_to_replace, replace_with, expcted_str_list):
-    result = crud_utils.replace_str_list(str_list_to_test, str_to_replace, replace_with)
-    if result:
-        for index, r in enumerate(result):
-            assert r == expcted_str_list[index]
-    else:
-        assert result == expcted_str_list
\ No newline at end of file
+# @pytest.mark.parametrize("str_list_to_test, str_to_replace, replace_with, expcted_str_list",[
+#     (['FAM_ACCESS_ADMIN', 'FOM_DEV_ACCESS_ADMIN', 'FOM_TEST_ACCESS_ADMIN'],
+#      "_ACCESS_ADMIN", "",
+#      ['FAM', 'FOM_DEV', 'FOM_TEST']
+#     ),
+#     (['FAM_ACCESS', 'FOM_DEV', 'FOM'],
+#      "_ACCESS", "_ACCESS_ADMIN",
+#      ['FAM_ACCESS_ADMIN', 'FOM_DEV', 'FOM']
+#     ),
+#     (None, "something", "some_other_thing", None)
+# ])
+# def test_replace_str_list(str_list_to_test, str_to_replace, replace_with, expcted_str_list):
+#     result = crud_utils.replace_str_list(str_list_to_test, str_to_replace, replace_with)
+#     if result:
+#         for index, r in enumerate(result):
+#             assert r == expcted_str_list[index]
+#     else:
+#         assert result == expcted_str_list
```

Comment:
These tests are the same for both postgres and sqlite tests, comment out in the sqlite tests to reduce duplicate.